src/Uno.UI/UI/Xaml/Documents/TextFormatting/FontDetails.skia.cs

```diff
@@ -1,10 +1,20 @@
-﻿using HarfBuzzSharp;
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using HarfBuzzSharp;
+using Microsoft.UI.Xaml.Controls;
 using SkiaSharp;
 
 namespace Microsoft.UI.Xaml.Documents.TextFormatting;
 
-internal record FontDetails(SKFont SKFont, float SKFontSize, float SKFontScaleX, SKFontMetrics SKFontMetrics, SKTypeface SKTypeface, Font Font, Face Face)
+internal record FontDetails(SKFont SKFont, float SKFontSize, float SKFontScaleX, SKFontMetrics SKFontMetrics, Font Font, bool CanChange)
 {
+	// TODO: Investigate best value to use here. SKShaper uses a constant 512 scale, Avalonia uses default font scale. Not 100% sure how much difference it
```

Comment:
@Youssef1313 do we have a task for this TODO?

Reply:
I think no, it's an existing comment but just moved around.